### PR TITLE
Clarify docs in spharm_functions.py

### PR DESCRIPTION
**Pull request by jammm**
I think this might make it a bit clearer to understand. If this is not acceptable, feel free to close this PR.

Thanks!

### CODE REVIEW

File: pyshtools/expand/spharm_functions.py [hunk 1]
```diff
@@ -218,7 +218,7 @@ def spharm_lm(l, m, theta, phi, normalization='4pi', kind='real', csphase=1,
         Condon-Shortley phase of (-1)^m will be appended to the spherical
         harmonic functions.
     degrees : optional, bool, default = True
-        If True, colat and phi are expressed in degrees.
+        If True, 'theta' and 'phi' are expressed in degrees.
```
Comment (MarkWieczorek) [inline pyshtools/expand/spharm_functions.py]: can you use back ticks for consistence (and also for use with the web documentation)?

**Comment by MarkWieczorek** [comment]
Hi. This looks good to me, but could you base this PR on the develop branch? We only use master for releases.

Also, could you use backticks for all variables that are used by the function? This is not only for consistency, but it makes a difference in how this is rendered for the html documentation. Thanks!

File: pyshtools/expand/spharm_functions.py (hunk 1 above)
Comment (jammm) [reply pyshtools/expand/spharm_functions.py]: Ah, I missed that. Thanks!

**Comment by jammm** [comment]
Closing PR and making a new one for `develop` branch.

**Pull request closed without merging**